test/e2e/kubedns/kubedns.go

```diff
@@ -55,4 +59,69 @@ var _ = Describe("kube-dns", func() {
 			kubeDNS.Stop()
 		})
 	})
+
+	It("should forward missed ptr lookup to upstream server", func() {
```

Comment:
should forward PTR queries to the upstream server

Reply:
Done.